### DISCUSSION TITLE
自定义ui，用svg或者其他icon，不使用@univerjs/icons How to use svg icons or other icons in custom ui instructions

### DISCUSSION THREAD

**Discussion by 13701138894** [dream-num/univer#3687] [Q&A | 常见问题]
![image](https://github.com/user-attachments/assets/42725e7f-d317-4381-ab0c-05431835036a)

**Comment by jikkai** [comment]
请勿重复提交问题，否则将被视为滥用

**Reply by 13701138894** [reply]
你全给关闭了，我不重复提交怎么办

**Reply by 13701138894** [reply]
你总得留一个open状态的吧

**Reply by jikkai** [reply]
https://github.com/dream-num/univer/discussions/3663
